### PR TITLE
Charter for Bazel-IntelliJ plug-in SIG.

### PR DESCRIPTION
**Pull request by raoarun**

### CODE REVIEW

**Comment by raoarun** [comment]
registered github account.

**Comment by raoarun** [comment]
@sventiffe please review the SIG charter pull request.  i will add content to it shortly.  thanks.

**Comment by raoarun** [comment]
@sventiffe please review the SIG charter pull request.  thank you.

**Comment by raoarun** [comment]
please add @jastice to the list of reviewers, @sventiffe ... thank you!

**Review by hicksjoseph — APPROVED** [review]

File: sigs/bazel-intellij/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  * Demos
+  * Monthly updates on new features and product roadmaps
+* Provide a platform for users and developers to regularly share knowledge and learn from each other
+* Connect and foster engagements across various engineering teams.
+
+The SIG will meet once a month.  Members can log issues / topics of interest in bazel-intellij@googlegroups.com.
+
+## Membership
+
+This SIG is open to anybody who uses, develops, or has an interest in sharing or absorbing knowledge on Bazel and the IntelliJ IDE plug-in. Please join bazel-intellij@googlegroups.com and share with interested colleagues.
```
Comment (sventiffe) [inline sigs/bazel-intellij/CHARTER.md]: I'd add a link to the actual group for simplicity (ie makes it easier to join and to view conversations)
[bazel-intellij@googlegroups.com](https://groups.google.com/g/bazel-intellij)

File: sigs/bazel-intellij/CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Resources
+
+* bazel-intellij@googlegroups.com
```
Comment (sventiffe) [inline sigs/bazel-intellij/CHARTER.md]: [bazel-intellij@googlegroups.com](https://groups.google.com/g/bazel-intellij)

**Review by sventiffe — CHANGES_REQUESTED** [review]
Charter LGTM, only two nits about link targets

**Comment by sventiffe** [comment]
> please add @jastice to the list of reviewers, @sventiffe ... thank you!

I've added Justin as collaborator to the repo, which is required before being able to review a PR. Access control is something we need to fine tune (@aiuto for awareness -- would it be an alternative to add an org level team to this repo so that every known collaborator could create a PR and be added as a reviewer?)

File: sigs/bazel-intellij/CHARTER.md (hunk 1 above)
Comment (raoarun) [reply sigs/bazel-intellij/CHARTER.md]: thanks @sventiffe

File: sigs/bazel-intellij/CHARTER.md (hunk 2 above)
Comment (raoarun) [reply sigs/bazel-intellij/CHARTER.md]: thanks @sventiffe

**Comment by raoarun** [comment]
@sventiffe i made the changes you requested.  ptal and lmk.  thanks!

**Review by sventiffe — APPROVED** [review]

**Review by sventiffe — APPROVED** [review]

**Comment by raoarun** [comment]
@sventiffe how do we merge this into main?

**Pull request merged**